<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1071</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1071</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AUFR</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/app_icon.jpg</normaloff>icon/app_icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>870</width>
      <height>560</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <widget class="QLabel" name="video_feed">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>870</width>
       <height>560</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>560</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>150</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>RECTANGLE</string>
     </property>
     <widget class="QRadioButton" name="face_rect_radio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Face</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">rect_radio_group</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="save_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>120</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>150</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>ACTIONS</string>
     </property>
     <widget class="QPushButton" name="generate_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>120</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Dataset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="training_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>120</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Train Model</string>
      </property>
     </widget>
     <widget class="QPushButton" name="extract_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>120</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Extract Features</string>
      </property>
     </widget>
     <widget class="QPushButton" name="new_user_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>New User</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>150</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>ALGORITHMS</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="lbph_algo_radio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>LBPH</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">algo_radio_group</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="fisher_algo_radio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Fisherfaces</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">algo_radio_group</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="eigen_algo_radio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>120</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Eignefaces</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">algo_radio_group</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>1051</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QProgressBar" name="progress_bar_recognize">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Confidence</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Trainined</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progress_bar_train">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progress_bar_generate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Generated</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_btn">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>15</y>
       <width>87</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>10</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icon/github.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>FACESEC</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>24</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>719</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="rect_radio_group"/>
  <buttongroup name="algo_radio_group"/>
 </buttongroups>
</ui>
